test/browser/spec/ol/control/attribution.test.js

```diff
@@ -31,6 +31,7 @@ describe('ol.control.Attribution', function () {
         new Attribution({
           collapsed: false,
           collapsible: false,
+          staticAttribution: 'static',
```

Comment:
Should I be adding another test to handle having a static attribution vs not having one. I think this is covered as part of the other tests, but I can add another if you think its needed.